interactors/badge.js

```diff
@@ -0,0 +1,11 @@
+import HTML from './baseHTML';
+
+const color = (element) => element.className;
+
+export default HTML.extend('badge')
+  .selector('[class^=badge-]')
+  .locator(element => element.querySelector('[class^=label-]').textContent)
+  .filters({
+    color,
+    value: (element) => element.querySelector('[class^=label-]').textContent
```

Comment:
Are you sure you want `textContent` here and not `innerText`? `textConent` will also include the content of script tags and stylesheets, as opposed to only those things which the browser renders as text.

Reply:
Ah.. yes - I like the emphasis on 'the text the browser renders' - it was in a handful of the interactors we had, so I went with it - I might have thought for a half-second to look up the difference. Good call!
Seems like it could be used/preferred across the board...

Reply:
Yeah, we converted all of ours to use `innerText` and haven't had any issue with it thus far.